platformsh/platformsh-client-js
# Use different eventsource polyfill

To test:
- Run this branch of the client in your local
- Go to any project overview page
- Check in the network tab that the /subscribe call is being done only once
- Push to that project to trigger an activity
Note!!! You won't be able to see the pings or the events in the event source preview on the network tab, as this is a polyfill and it comes with this disadvantage, you can preview them by adding an event handler and a console log
